src/raiden_libs/user_address.py

```diff
@@ -94,6 +95,9 @@ def known_addresses(self) -> Set[Address]:
         # may be modified while these values are used. Issue: #5240
         return set(self._address_to_userids)
 
+    def seen_offline_at(self, address: Address) -> datetime:
+        return self._first_seen_offline.get(address, datetime.fromtimestamp(0))
```

Comment:
At a later point (or maybe right now?) we will want to make the default return value the timestamp of the time the PFS came online. That way we won't convey incorrect age information on unknown (to the PFS) users. In detail:
right now a user that has not been seen by the PFS will be considered offline since 30+ years, while all the PFS knows is an offline time since it was started.

Reply:
Let me reword this:
We want to return
```
(datetime.now() - PathFindingService.started_at).seconds
```
where `PathFindingService.started_at` will just be set on `PathFindingService.__init__()` to `datetime.now()`

Reply:
...or the analogous considering https://github.com/raiden-network/raiden-services/pull/998#discussion_r630931188